packages/flutter_tools/lib/src/device.dart

```diff
@@ -276,6 +276,10 @@ abstract class Device {
   /// Whether this device implements support for hot restart.
   bool get supportsHotRestart => true;
 
+  /// Whether flutter applications running on this device can be terminated
+  /// from the vmservice.
+  bool get supportsStopApp => true;
```

Comment:
Should the 'supportsX' fields be pulled out to a separate class like "DeviceCapabilities"?

Reply:
Sounds like a good idea, done